tests/cmds/test_users.py

```diff
@@ -404,6 +396,43 @@ def test_update_user_calls_update_user_with_correct_parameters_when_all_are_pass
     )
 
 
+def test_update_when_py42_raises_invalid_email_outputs_error_message(
```

Comment:
New tests to make sure the newly added py42 errors work where they are supposed to (meaning, the output looks nice when these get raised instead of just guiding the user to investigate the logs)